strutils/utils_test.go

```diff
@@ -20,27 +19,29 @@ var checkUTF8LenTests = []struct {
 	{[]byte{0xf0, 0b10010000, 0b10011000, 0b10011100}, 4}, // s5
 	{[]byte{0xf1, 0b10000010, 0b10100000, 0b10101010}, 4}, // s6
 
-	{[]byte{0xa9}, 4}, // xx
+	{[]byte{0xa9}, 1}, // xx
 
-	{[]byte{}, 1}, // line 6
+	{[]byte{}, 0}, // line 6
 
 	// add test for p[1] < accept.lo || accept.hi < p[1]
-	{[]byte{0xc2, 0b00110001}, 2}, // line 22
-	{[]byte{0xc2, 0b11111111}, 2}, // line 22
+	{[]byte{0xc2, 0b00110001}, 1}, // line 22
```

Comment:
Why there are two tests for a case

Reply:
one for` p[1] < accept.lo` and another for `accept.hi < p[1]`